src/vnc.yml

```
@@ -4,7 +4,21 @@
   become: yes
   become_method: sudo
   roles:
-    - vnc
+    - role: vnc
+      vars:
+        # The user information and ssh keys for the VNC user
+        password: "{{ lookup('aws_ssm', '/vnc/password') }}"
+        private_ssh_key: "{{ lookup('aws_ssm', '/vnc/ssh/rsa_private_key') }}"
+        public_ssh_key: "{{ lookup('aws_ssm', '/vnc/ssh/rsa_public_key') }}"
+        user_groups:
```

Shouldn't we be including the username here?
```suggestion
        public_ssh_key: "{{ lookup('aws_ssm', '/vnc/ssh/rsa_public_key') }}"
        username: "{{ username }}"
        user_groups:
```

---

No, the scope of the `username` variable at the bottom of this file has sufficient scope that it gets picked up here too.

---

Ah right, I forgot about that.  Thanks for the reminder!